README.md

```diff
@@ -304,13 +304,13 @@ npm run start
 
 And you should have the same content in the main browser window as before once this completes. But if you open up your inspector using `cmd+option+i` or `ctrl+shift+c` you will see this inside of your console:
 
-![[Screenshot 2023-10-24 at 4.45.49 PM.png]]
+![Console Output](image.png)
```

Comment:
i created an assets folder that i will be merging in, but you can just create one as well and put it at the root directory of the project